index.html

```diff
@@ -1477,13 +1477,18 @@ <h6>Item-specific Language</h6>
 								community, introduces such a feature, future versions of this specification may extend
 								the ability of manifests to include this.</p>
 
-							<p>When using manifests with bidirectional text, user agents SHOULD identify the base
-								direction of any given natural language value by scanning the text for the first strong
-								directional character. Once the base direction has been identified, user agents MUST
-								determine the appropriate rendering and display of natural language values according to
-								the Unicode Bidirectional Algorithm&#160;[[!bidi]]. This could require wrapping
-								additional control characters or markup around the string prior to display, in order to
-								apply the base direction. (See <a href="#app-bidi-examples"></a>.)</p>
+
+							<p>
+								In order to correctly handle manifests entries containing right-to-left or bidirectional text, user agents SHOULD identify the base direction of any given natural language value by scanning the text for the first strong directional character.
+							</p>
+
+							<p class=note>
+								In situations where the first-strong heuristics will produce the wrong result (eg. a string in the Arabic or Hebrew script that begins with a Latin acronym), content developers may want to prepend a Unicode formatting character to the string. This would then produce the necessary base direction when the heuristics are applied. They should use one of the following formatting characters: U+200E LEFT-TO-RIGHT MARK, or U+200F RIGHT-TO-LEFT MARK.
+							</p>
+							
+							<p>
+								Once the base direction has been identified, user agents MUST determine the appropriate rendering and display of natural language values according to the Unicode Bidirectional Algorithm[[!bidi]]. This could require wrapping additional control characters or markup around the string prior to display, in order to apply the base direction. (See <a href="#app-bidi-examples"></a>.	
```

Comment:
per Najib's suggestion in , this could be changed as follows:
"additional control characters or markup"
->
"additional markup or control characters"

Reply:
Added the change in https://github.com/w3c/wpub/pull/442/commits/5d614efcf7cb75c0e8c80e9620bfad85aad84eed